sys/src/cmd/acid/builtin.c

```diff
@@ -136,7 +136,6 @@ dosysr1(Node *r, Node*_)
 	r->op = OCONST;
 	r->type = TINT;
 	r->fmt = 'D';
-	fprint(2, "Thank Giacomo -- no sysr1 -- we'll bring it back\n");
```

Comment:
either verify that sysr1 is working again (it should be) or put an error and panic here. And, this should be a different PR.